### PR TITLE
Fix test compatibility with zope.i18n 4.3

### LINKED ISSUE #8: import errors in zope.browserresource.tests.test_i18nfile
I run tests by ```python setup.py test -v```
and faced an error:
```
...
runTest (zope.testrunner.find.StartUpFailure)
StartUpFailure: import errors in zope.browserresource.tests.test_i18nfile. ... ERROR
test (zope.browserresource.tests.test_icondirective.Test) ... ok
...
======================================================================
ERROR: runTest (zope.testrunner.find.StartUpFailure)
StartUpFailure: import errors in zope.browserresource.tests.test_i18nfile.
----------------------------------------------------------------------
```
```python
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/zope/testrunner/find.py", line 149, in runTest
    six.reraise(*self.exc_info)
  File "/usr/src/RPM/BUILD/python-module-zope.browserresource-4.2.1/src/zope/browserresource/tests/test_i18nfile.py", line 40, in <module>
    from zope.i18n.tests.testii18naware import TestII18nAware
ImportError: No module named testii18naware
```
```
----------------------------------------------------------------------
Ran 47 tests in 0.202s

FAILED (errors=1)
Test failed: <unittest.runner.TextTestResult run=47 errors=1 failures=0>
error: Test failed: <unittest.runner.TextTestResult run=47 errors=1 failures=0>

```
After [ac16be8fed6cc840ad0f3244c883c8b7a672f643](https://github.com/zopefoundation/zope.i18n/commit/ac16be8fed6cc840ad0f3244c883c8b7a672f643) there is no more 
testiii18naware.py within i18n.

### PR DESCRIPTION
**Pull request by jamadden**
Fixes #8

Also fix whitespace warnings in test_i18nfile.py and remove unneeded use of test_suite (since we're not importing TestI18NAware anymore).

### CODE REVIEW

File: src/zope/browserresource/tests/test_i18nfile.py [hunk 1]
```diff
@@ -37,10 +37,33 @@
...
 from zope.i18n.negotiator import negotiator
 
-from zope.i18n.tests.testii18naware import TestII18nAware
 
 test_directory = os.path.dirname(p.__file__)
 
+class AbstractTestII18nAwareMixin(object):
+
+
+    def setUp(self):
```
Comment (mgedmin) [inline src/zope/browserresource/tests/test_i18nfile.py]: Too many blank lines here for my taste (and I'm sure flake8 would agree with me ;).

File: src/zope/browserresource/tests/test_i18nfile.py [hunk 2]
```diff
@@ -37,10 +37,33 @@
...
+    def testSetDefaultLanguage(self):
+        self.object.setDefaultLanguage('lt')
+        self.assertEqual(self.object.getDefaultLanguage(), 'lt')
+
+    def testGetAvailableLanguages(self):
+        self.assertEqual(sorted(self.object.getAvailableLanguages()), ['en', 'fr', 'lt'])
+
+
 
 @adapter(IFileResource, IBrowserRequest)
```
Comment (mgedmin) [inline src/zope/browserresource/tests/test_i18nfile.py]: One too many blank lines here also (should be two, not three, between top-level classes).

(Unless you did it to be consistent with the rest of the file, which is commendable.)

File: src/zope/browserresource/tests/test_i18nfile.py (hunk 1 above)
Comment (jamadden) [reply src/zope/browserresource/tests/test_i18nfile.py]: That's a result of copy/paste from a diff. Honestly, I did notice the extra lines before I committed...but I really just wanted to see if you would comment on them 😉

**Comment by jamadden** [comment]
Thanks!

**Pull request merged**